index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+
+  <body>
+    <header>
+      <a href="" class="logo">My Logo</a>
+      <nav>
+        <div class="nav-icon">
+          <a href="#"> <i class="fa fa-solid fa-bars"></i></a>
+        </div>
+      </nav>
+    </header>
+    <section class="main">
+      <h1 class="h1">I'm Ezechukwu moses Glad to see you!</h1>
+      <p class="p">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#" class="h4-connect">LET'S CONNECT</a>
+
+      <ul class="flex item-center">
+        <li class="icon-tag">
+          <a href="#"><img src="./image/twitter.png" alt="twitter" /></a>
+        </li>
+        <li class="icon-tag">
+          <a href="#"><img src="./image/linkedin.png" alt="linkedin" /></a>
```

Comment:
- [ ] As demanded by the previous reviewer, kindly fix the link to the LinkedIn icon. I notice the icon name is capitalised, consider renaming the icon to match the link here.

Reply:
ok will adjust it